db/schema.rb

```diff
@@ -0,0 +1,18 @@
+# This file is auto-generated from the current state of the database. Instead
```

Comment:
Is this related to Travis? If not is it worth removing it from this PR (eg. using `git commit --amend`) and commit it separately?

Reply:
it is done :)